### PR TITLE
Changed About.js to Brand.js and similar changes

### LINKED ISSUE #1: Rename About.js to Brand.js
Pm asked to change the nav item to say "Brands" instead of "About Us." It would be nice to change the code to reflect this

### PR DESCRIPTION
**Pull request by rahul-s-bhatt**

### CODE REVIEW

File: src/pages/about.js [hunk 1]
```diff
@@ -1,34 +0,0 @@
-import React from 'react';
```
Comment (JameelJiwani) [inline src/pages/about.js]: This file should have been renamed to `brand.js`not deleted

File: src/routes.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 import ContactPage from './pages/contact';
-import AboutPage from './pages/about';
+import BrandPage from './pages/brand';
```
Comment (JameelJiwani) [inline src/routes.js:5]: Nice work!

**Review by JameelJiwani — CHANGES_REQUESTED** [review]
good work. Almost there 😀

File: src/pages/about.js (hunk 1 above)
Comment (rahul-s-bhatt) [reply src/pages/about.js]: Yeah I renamed "about.js" to "brand.js" and then did all the changes.
Did I go wrong somewhere?

File: src/routes.js (hunk 2 above)
Comment (rahul-s-bhatt) [reply src/routes.js:5]: Thank you!

**Review by JameelJiwani — CHANGES_REQUESTED** [review]
Have you tested this locally? I think you need to change 1 more thing inside of NavLinks.js

**Review by rahul-s-bhatt — COMMENTED** [review]
> Have you tested this locally? I think you need to change 1 more thing inside of NavLinks.js

yeah changed it

**Review by JameelJiwani — APPROVED** [review]

**Pull request merged**